server/db/driver/pg/matches_online_test.go

```diff
@@ -1327,6 +1356,41 @@ func TestEpochReport(t *testing.T) {
 		QuoteVolume: 100,
 	})
 
+	startStamp = uint64((epochIdx + 2) * epochDur)
+	endStamp = startStamp + uint64(epochDur)
+	candle = &candles.Candle{
+		StartStamp:  startStamp,
+		EndStamp:    endStamp,
+		MatchVolume: 100,
+		HighRate:    100,
+		LowRate:     100,
+		StartRate:   100,
+		EndRate:     100,
+		QuoteVolume: 100,
+	}
+	addCandles[2] = candle
+
+	startStamp = uint64((epochIdx + 2) * epochDur)
+	endStamp = startStamp + uint64(epochDur)
+	dayCandle := &candles.Candle{
+		StartStamp:  startStamp,
+		EndStamp:    endStamp,
+		MatchVolume: 112,
+		HighRate:    100,
+		LowRate:     3,
+		StartRate:   4,
+		EndRate:     100,
+		QuoteVolume: 122,
+	}
+
+	if err = archie.InsertCandles(42, 0, uint64(epochDur), addCandles); err != nil {
+		t.Fatalf("error inserting candles: %v", err)
+	}
+
+	if err = archie.InsertCandles(42, 0, uint64(time.Hour*24/time.Millisecond), []*candles.Candle{dayCandle}); err != nil {
+		t.Fatalf("error inserting candle: %v", err)
+	}
+
```

Comment:
I guess this hasn't worked in a while, since we don't add candles automatically, it seems. This is done in server/apidata/apidata.go now.

Unsure why the test is just now failing.